src/DateTimePicker.jsx

```diff
@@ -132,6 +132,7 @@ class DateTimePicker extends React.Component {
 	*/
 	setDateTime(value) {
 		this.setState({ datetime: new Date(value) });
+		this.props.onChangeCallback(this.state.datetime, this.props.name);
```

Comment:
since `onChangeCallback` is an optional prop, you'll need to check that it exists before calling it.